wandb/sdk/interface/interface_grpc.py

```diff
@@ -270,6 +270,20 @@ def _communicate_poll_exit(
             ret = pb.PollExitResponse()
         return ret
 
+    def _publish_keepalive(self, keepalive: pb.KeepaliveRequest) -> None:
+        assert self._stub
+        self._assign(keepalive)
+        _ = self._stub.Keepalive(keepalive)
+        return None
+
+    def _communicate_server_info(
+        self, server_info: pb.ServerInfoRequest
+    ) -> Optional[pb.ServerInfoResponse]:
+        assert self._stub
+        self._assign(server_info)
+        ret = self._stub.ServerInfo(server_info)
+        return ret
```

Comment:
why is it not covered?! I don't understand ...

Reply:
Because we use deliver_server_info now.... the communicate verb will be removed in an upcoming phase.